annotation/tests/test_jobs.py

```diff
@@ -48,113 +41,87 @@ def categories():
 
 
 @pytest.fixture
-def jobs_to_test_progress(job_annotators: Tuple[User], categories: Category):
+def jobs_to_test_progress(users: Tuple[User], categories: Category):
     yield (
         Job(
             job_id=1,
             name="test1",
             callback_url="http://www.test.com",
-            annotators=[job_annotators[0], job_annotators[1]],
+            annotators=[users[0], users[1]],
             validation_type=ValidationSchema.cross,
             is_auto_distribution=False,
             categories=[categories],
             deadline="2021-10-19T01:01:01",
             tenant="test",
+            extensive_coverage=1,
         ),
         Job(
             job_id=2,
             name="test2",
             callback_url="http://www.test.com",
-            annotators=[job_annotators[0], job_annotators[1]],
-            validation_type=ValidationSchema.cross,
+            annotators=[users[0], users[1]],
+            validation_type=ValidationSchema.validation_only,
             is_auto_distribution=False,
             categories=[categories],
             deadline="2021-10-19T01:01:01",
             tenant="test",
+            extensive_coverage=1,
         ),
     )
 
 
 @pytest.fixture
 def tasks():
+    status = (
+        TaskStatusEnumSchema.finished,
+        TaskStatusEnumSchema.in_progress,
+        TaskStatusEnumSchema.ready,
+        TaskStatusEnumSchema.pending,
+        TaskStatusEnumSchema.ready,
+        TaskStatusEnumSchema.pending,
+    )
+    tasks = []
+    for i in range(1, 7):
+        tasks.append(
+            ManualAnnotationTask(
+                id=i,
+                file_id=i if i != 6 else 1,
+                pages=[1] if i != 5 else [3],
+                job_id=i if i < 4 else 10,
+                user_id="7b626e68-857d-430a-b65b-bba0a40417ee",
+                is_validation=False if i < 5 else True,
+                status=status[i - 1],
+                deadline=None,
```

Comment:
you could split that into multiple fixtures really, since we dont have much time lets keep it like that